langchain4j-milvus/src/main/java/dev/langchain4j/store/embedding/milvus/MilvusEmbeddingStore.java

```diff
@@ -295,6 +333,12 @@ public static class Builder {
         private String metadataFieldName;
         private String vectorFieldName;
 
+
+        public Builder milvusClient(MilvusServiceClient milvusClient) {
```

Comment:
```suggestion
        public Builder milvusClient(MilvusClient milvusClient) {
```
Should we accept `MilvusClient` instead? There seems to be multiple implementations in milvus SDK

Reply:
@langchain4j 

do you really need to cahnge the all to MilvusClient ?

because all of the places is


![image](https://github.com/user-attachments/assets/e7b970f7-2a8c-409d-9505-03f511785790)

Reply:
ok, let's keep `MilvusServiceClient`, we can always change it to `MilvusClient` if it will be required